proposals/PluginsAudit2015/audit.md

```diff
@@ -0,0 +1,223 @@
+# Cordova Plugins
+
+**NOTE**: Data for this proposal was collected on October 28th, 2015. 
+
+## Battery-Status
+* 1 Month Downloads: 5116
+* Open Issues: 18
+* Open Pull Requests: 3
+* Spec: http://www.w3.org/TR/battery-status/ 
+* android 5 has the ability to grab battery info in uiwebview built in
+* isPlugged is useful but not being used much
```

Comment:
Just curious: how do we know which one of these API endpoints are used a lot and which ones aren't ?